library/cirros

```diff
@@ -1,4 +1,5 @@
 # maintainer: Eric Windisch <ewindisch@docker.com> (@ewindisch)
 
-latest: git://github.com/ewindisch/docker-cirros@1cded459668e8b9dbf4ef976c94c05add9bbd8e9
+latest: git://github.com/ewindisch/docker-cirros@5ef3f5024b0aa80553cc34be9eff6685cb31b458
+0.3.3: git://github.com/ewindisch/docker-cirros@5ef3f5024b0aa80553cc34be9eff6685cb31b458
 0.3.0: git://github.com/ewindisch/docker-cirros@1cded459668e8b9dbf4ef976c94c05add9bbd8e9
```

Comment:
That's fine & reasonable. I'll remove 0.3.0 then.

I had seen other official images keeping clearly-not-supported versions tagged here, so this wasn't clear. I'll follow-up on that and see about having those images remove their unsupported versions, too.